pages/onboarding_new_cluster.ipynb

```diff
@@ -0,0 +1,826 @@
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Writing 'text_input' (str) to file 'keycloak/overlays/moc/infra/clients/my-cluster.yaml'.\n",
+      "[PGP]\t \u001b[33mWARN\u001b[0m[0000] Deprecation Warning: GPG key fetching from a keyserver within sops will be removed in a future version of sops. See https://github.com/mozilla/sops/issues/727 for more information. \n"
+     ]
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Encrypted files are listed in `secret-generator` files, which for this scope is located in `keycloak/overlays/$CLUSTER_REGION/infra/secret-generator.yaml`. \n",
+    "```\n",
+    "---\n",
+    "apiVersion: viaduct.ai/v1\n",
+    "kind: ksops\n",
+    "metadata:\n",
+    "    name: secret-generator\n",
+    "files:\n",
+    "    ...\n",
+    "    - clients/$CLUSTER_NAME.enc.yaml\n",
+    "    ...\n",
+    "```\n",
+    "Create `operate-first-sso-secret` file in location `cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/oauths/operate-first-sso-secret.yaml`."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "source": [
+    "!mkdir -p cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/oauths/\n",
+    "text_input=\"\"\"\\\n",
+    "apiVersion: v1\n",
+    "kind: Secret\n",
+    "metadata:\n",
+    "    name: operate-first-sso-secret\n",
+    "    namespace: openshift-config\n",
+    "    annotations:\n",
+    "        argocd.argoproj.io/compare-options: IgnoreExtraneous\n",
+    "        argocd.argoproj.io/sync-options: Prune=false\n",
+    "type: Opaque\n",
+    "stringData:\n",
+    "    clientSecret: $CLUSTER_KEYCLOAK_SECRET\n",
+    "\"\"\"\n",
+    "%store text_input >cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/oauths/operate-first-sso-secret.yaml\n",
+    "!sops --encrypt --encrypted-regex=\"^clientId\" --pgp=\"0508677DD04952D06A943D5B4DC4116D360E3276\" cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/oauths/operate-first-sso-secret.yaml >cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/oauths/operate-first-sso-secret.enc.yaml\n",
+    "!rm cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/oauths/operate-first-sso-secret.yaml"
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Writing 'text_input' (str) to file 'cluster-scope/overlays/prod/moc/my-cluster/oauths/operate-first-sso-secret.yaml'.\n",
+      "[PGP]\t \u001b[33mWARN\u001b[0m[0000] Deprecation Warning: GPG key fetching from a keyserver within sops will be removed in a future version of sops. See https://github.com/mozilla/sops/issues/727 for more information. \n"
+     ]
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Create `cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/secret-generator.yaml`."
```

Comment:
```suggestion
    "Since this is an encrypted secret. We need to tell Kustomize how to access it and how to decrypt it. This means we need yet another `secret-generator.yaml`, similar to what we created above. Create `cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/secret-generator.yaml`."
```

Reply:
Resolved in latest commit.